packages/react/src/Image/Image.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import clsx from 'clsx'
+import styles from './Image.module.css'
+
+type AspectRatio = '1:1' | '16:9' | '16:10' | '4:3' | 'custom'
+
+export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  ref?: React.Ref<HTMLImageElement>
+  alt: string
+  as?: 'picture' | 'img'
+  aspectRatio?: AspectRatio
+}
+
+const aspectRatioResolver = (ratio?: AspectRatio) => {
+  if (ratio === undefined) return ''
```

Comment:
Is this part necessary? As you're checking for truthy occurrence of `aspectRatio`,  so this may be redundant.

Reply:
I don't believe so I will remove it.